safe/gui/tools/wizard/step_fc00_functions1.py

```diff
@@ -119,40 +119,35 @@ def populate_function_table_1(self):
         self.tblFunctions1.setRowCount(len(exposures))
         for i in range(len(hazards)):
             hazard = hazards[i]
-            item = QtGui.QTableWidgetItem()
-            item.setIcon(QtGui.QIcon(
-                resources_path('img', 'wizard', 'keyword-subcategory-%s.svg'
-                               % (hazard['key'] or 'notset'))))
+            item = QTableWidgetItem()
+            item.setIcon(QIcon(get_image_path(hazard)))
             item.setText(hazard['name'].capitalize())
+            item.setTextAlignment(Qt.AlignLeft)
             self.tblFunctions1.setHorizontalHeaderItem(i, item)
         for i in range(len(exposures)):
             exposure = exposures[i]
-            item = QtGui.QTableWidgetItem()
-
-            item.setIcon(QtGui.QIcon(resources_path(
-                'img', 'wizard', 'keyword-subcategory-%s.svg'
-                % (exposure['key'] or 'notset'))))
+            item = QTableWidgetItem()
+            item.setIcon(QIcon(get_image_path(exposure)))
             item.setText(exposure['name'].capitalize())
             self.tblFunctions1.setVerticalHeaderItem(i, item)
         developer_mode = setting('developer_mode', False, bool)
         for hazard in hazards:
             for exposure in exposures:
-                item = QtGui.QTableWidgetItem()
+                item = QTableWidgetItem()
                 if (exposure in hazard['disabled_exposures'] and not
                         developer_mode):
                     background_colour = unavailable_option_color
                     # Set it disable and un-selectable
                     item.setFlags(
                         item.flags() & ~
-                        QtCore.Qt.ItemIsEnabled & ~
-                        QtCore.Qt.ItemIsSelectable
+                        Qt.ItemIsEnabled & ~
```

Comment:
First time I see this bitwise operator in python

Reply:
:D